docker-compose-local.yml

```diff
@@ -69,6 +69,7 @@ services:
       - CP_EXTERNAL_USER_NAME
       - CP_EXTERNAL_PASSWORD
       - CP_INTERNAL_PASSWORD
+      - EXTERNAL_SERVICE_BASIC_AUTHORISATION_WHITELIST
```

Comment:
This new env variable should be added in Jenkins files & values.yaml as well.

Reply:
hey Hemanta, I've already added it to `values.yaml`.
I have skipped jenkinsfile since we don't have other variables there. Is it because we don't have functional tests for the gateway atm?
but I can add it anyway to jenkinsfile 👍